authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,734 @@
+When xref:#sts-mode-installing[installing] an {product-title} cluster that is configured to use the CCO in manual mode with STS, you can use the `sts-preflight` tool to create the required AWS resources.
+
+This tool, which is hosted in an external repository, is provided to help configure a cluster that uses STS mode while the feature is a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+
+As an alternative, you can xref:#sts-mode-installing-manual-config[create the required AWS resources manually].
+
+[id="sts-mode-installing-preflight-preparing"]
+==== Preparing the installer
+
+.Prerequisites
+* Obtain the pull secret for your cluster.
+* Obtain the {product-title} version 4.7 installation binary.
+
+.Procedure
+
+. From the directory that contains the installation program, obtain details of the {product-title} release image that your `openshift-install` binary is built to use:
++
+[source,terminal]
+----
+$ ./openshift-install version
+----
++
+.Example output
+[source,terminal]
+----
+./openshift-install 4.7.0
+built from commit 2d8b61dd8802058ffffcdc576e0cff67acebdb85
+release image registry.svc.ci.openshift.org/ocp/release@sha256:a322ab4941701452f428f5c138efb5354ad2aec33f6f7693f0a3033bfe257d8c
+----
+
+. Extract the AWS `CredentialsRequest` custom resources (CRs) from the release image:
++
+[source,terminal]
+----
+$ mkdir credreqs ; oc adm release extract --cloud=aws --credentials-requests quay.io/openshift-release-dev/ocp-release:4.<y>.<z>-x86_64 --to=./credreqs ; cat ./credreqs/*.yaml > credreqs.yaml
+----
++
+Where `<y>` and `<z>` are the numbers corresponding to the version of {product-title} you are installing.
+
+. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config
+----
+
+. Configure the cluster to install with the CCO in manual mode:
++
+[source,terminal]
+----
+$ echo "credentialsMode: Manual" >> install-config.yaml
+----
+
+. Create install manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests
+----
+
+[id="sts-mode-installing-preflight-using"]
+==== Using the sts-preflight tool
+
+.Prerequisites
+* Download the link:https://github.com/sjenning/sts-preflight[`sts-preflight` tool].
+
+.Procedure
+
+. Using an AWS account with administrator-level permissions, run the following command to:
++
+--
+** Generate public and private `ServiceAccount` object signing keys.
+** Create an S3 bucket.
+** Upload the OIDC config into the S3 bucket.
+** Set up an IAM Identity Provider that trusts the S3 bucket.
+** Create an IAM role with administrator rights to use for installation.
+** Create IAM roles for each AWS `CredentialsRequest` CR.
++
+[source,terminal]
+----
+$ ./sts-preflight create --infra-name <aws-infra-name> --region <aws-region> --credentials-requests-to-roles <path-to-credreqs.yaml>
+----
++
+Where `<aws-infra-name>` is a prefix the `sts-preflight` tool uses when naming the objects that are created in AWS.
```

Comment:
AWS folks likely know what the `aws-infra-name` is. Should we mention how to find it anyway?

Reply:
From what I understood, it's just a generated prefix. So later when they see it in e.g. `<aws-infra-name>-openshift-image-registry-installer-cloud-credentials` they would recognize "oh, it's that prefix"